src/components/TopAppBar/components/Logo.style.ts

```diff
@@ -5,11 +5,9 @@ import { flexCenter } from '../../../theme/functions';
 const wrapper = (theme: Theme): SerializedStyles => css`
   ${flexCenter};
   background-color: transparent;
-  margin: ${theme.spacing.md} ${theme.spacing.md} ${theme.spacing.md} 0px;
+  margin: ${theme.spacing.md} 72px ${theme.spacing.md} ${theme.spacing.md};
```

Comment:
This is outside the scope of this PR. 

Maybe we need more sizes in spacing to accommodate this and everything else that might use it.

Reply:
I agree this should be contacted with the design team and add a new larger spacing to cover this and other related cases.